build/check-style.sh

```diff
@@ -3,6 +3,7 @@
 set -euo pipefail
 
 export PKG=${PKG:-./...}
+export TAGS=${1:-"''"}
```

Comment:
And why isn't this `${1:-}`? You're quoting the expansion below so extra quotes shouldn't be needed.